src/develop/components/ColumnDoc.js

```diff
@@ -0,0 +1,157 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+var React = require('react');
+var jsxToString = require('jsx-to-string');
+var DocsArticle = require('../../DocsArticle');
+var OneColumn = require('../../modules/OneColumn');
+var TwoColumn = require('../../modules/TwoColumn');
+var ColumnContent = require('../../modules/ColumnContent');
+var DownloadIcon = require('grommet/components/icons/base/Download');
+
+OneColumn.displayName = 'OneColumn';
+
+var inline = "<{Number}Column />";
```

Comment:
Ok, I've split up the column modules into separate doc pages for OneColumn, TwoColumn, ThreeColumn, and FourColumn.